src/Features/Core/Portable/NavigateTo/AbstractNavigateToSearchService.InProcess.cs

```diff
@@ -74,16 +57,6 @@ internal abstract partial class AbstractNavigateToSearchService
             // Then process non-priority documents.
             var lowPriDocs = project.Documents.Where(d => !highPriDocs.Contains(d)).ToSet();
             await ProcessDocumentsAsync(searchDocument, patternName, patternContainerOpt, declaredSymbolInfoKindsSet, onResultFound, lowPriDocs, cancellationToken).ConfigureAwait(false);
-
-            // if the caller is only searching a single doc, and we already covered it above, don't bother computing
-            // source-generator docs.
-            if (searchDocument != null && (highPriDocs.Contains(searchDocument) || lowPriDocs.Contains(searchDocument)))
-                return;
-
-            // Finally, generate and process and source-generated docs.  this may take some time, so we always want to
-            // do this after the other documents.
-            var generatedDocs = await project.GetSourceGeneratedDocumentsAsync(cancellationToken).ConfigureAwait(false);
-            await ProcessDocumentsAsync(searchDocument, patternName, patternContainerOpt, declaredSymbolInfoKindsSet, onResultFound, generatedDocs.ToSet<Document>(), cancellationToken).ConfigureAwait(false);
```

Comment:
the normal SearchProject doesn't process generated docs anymore.  those are explicitly asked for at the end by the nav-to orchestrator.